core/src/ecs/Component.ts

```diff
@@ -1,12 +1,12 @@
-import { Actions, Clickable, Collider, Controlling, Data, Debug, Effects, Expires, Gun, Health, Input, NPC, Networked, Player, Position, Renderable, Team, entries, keys } from "@piggo-gg/core";
+import { Actions, Clickable, Collider, Controlling, Data, Debug, Effects, Expires, Gun, Health, Input, NPC, Networked, Player, Position, Renderable, Team, XY, entries, keys } from "@piggo-gg/core";
 
 export type ComponentTypes =
   Actions | Clickable | Collider | Controlling |
   Data | Debug | Effects | Expires | Health |
   Input | Team | Networked | NPC | Player |
   Position | Renderable | Gun
 
-export type NetworkedComponentData = Record<string, boolean | string | number | string[] | number[]>
+export type NetworkedComponentData = Record<string, boolean | string | number | string[] | number[] | XY>
```

Comment:
TODO `Record<string` here means that arbitrary keys are valid (removes type safety on `component.data.<key>` access

Reply:
done